Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.yml

```diff
@@ -0,0 +1,210 @@
+    description: Retrieves information about the specified IAM user, including the user creation date, path, unique ID, and ARN.
+    name: aws-iam-identitycenter-get-user
+    outputs:
+    - contextPath: AWS.IAM.IdentityCenter.Users.UserName
+      description: The friendly name identifying the user.
+      type: string
+    - contextPath: AWS.IAM.IdentityCenter.Users.UserId
+      description: The stable and unique string identifying the user.
+      type: string
+    - contextPath: AWS.IAM.IdentityCenter.Users.Email
+      description: The user email address.
+      type: string
+    - contextPath: AWS.IAM.IdentityCenter.Users.DisplayName
+      description: The user display name in AWS IAM IdentityCenter.
+  - arguments: []
+    description: Lists the IAM users, returns all users in the AWS account.
+    name: aws-iam-identitycenter-list-users
+    outputs:
+    - contextPath: AWS.IAMIdentityCenter.Users.UserName
+      description: The friendly name identifying the user.
+      type: string
+    - contextPath: AWS.IAMIdentityCenter.Users.UserId
+      description: The stable and unique string identifying the user.
+      type: string
+  - arguments: []
+    description: Lists all the IAM groups in the AWS account.
+    name: aws-iam-identitycenter-list-groups
+    outputs:
+    - contextPath: AWS.IAM.IdentityCenter.Groups.GroupName
+      description: The friendly name that identifies the group.
+      type: string
+    - contextPath: AWS.IAM.IdentityCenter.Groups.GroupId
+      description: The stable and unique string identifying the group.
+      type: string
+  - arguments:
+    - description: The name of the user to list groups for.
+      name: userName
+      required: true
+    description: Lists the IAM groups that the specified IAM user belongs to.
+    name: aws-iam-identitycenter-list-groups-for-user
+    outputs:
+    - contextPath: AWS.IAM.IdentityCenter.Users.GroupMemeberships.GroupName
+      description: The friendly name that identifies the group.
+      type: string
+    - contextPath: AWS.IAM.IdentityCenter.Users.GroupMemeberships.GroupId
+      description: The stable and unique string identifying the group.
+      type: string
+  - arguments:
+    - description: The name of the user to add.
+      name: userName
+      required: true
+    - description: The name of the group to update.
+      name: groupName
+      required: true
+    description: Adds the specified user to the specified group.
+    name: aws-iam-identitycenter-add-user-to-group
+  - arguments:
+    - name: groupName
+      required: true
+    description: The name of the group to search.
+    name: aws-iam-identitycenter-get-group
+  - arguments:
+    - description: Username that will be removed from all groups.
+      name: userName
+      required: true
+    description: This will remove the entered user from all groups/memberships.
+    name: aws-iam-identitycenter-remove-user-from-all-groups
+  - arguments:
+    - name: emailAddress
```

Comment:
Add a description here for this argument

Reply:
Done